integrations/github-cloud/github_integration/__init__.py

```
@@ -0,0 +1,4 @@
+from .client import *
```

Ruff (0.11.9)

`from .client import *` used; unable to detect undefined names

(F403)